{"name": "test_get_projects_paginated", "status": "failed", "statusDetails": {"message": "AssertionError: Number of projects are not equal to maxResults\nassert 7 == 5\n +  where 7 = len({'isLast': False, 'maxResults': 4, 'nextPage': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=4', 'self': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=0', ...})\n +    where {'isLast': False, 'maxResults': 4, 'nextPage': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=4', 'self': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=0', ...} = <bound method Response.json of <Response [200]>>()\n +      where <bound method Response.json of <Response [200]>> = <Response [200]>.json", "trace": "jira_api = <utils.jira_project_api.JiraProjectAPI object at 0x000001E6AF64C950>\n\n    def test_get_projects_paginated(jira_api):\n        StartAt=0\n        maxResults=5\n        response=jira_api.get_projects_paginated(StartAt,maxResults)\n        assert response is not None, \"test_get_projects_paginated: Unable to fetch projects\"\n>       assert len(response.json())==5, \"Number of projects are not equal to maxResults\"\nE       AssertionError: Number of projects are not equal to maxResults\nE       assert 7 == 5\nE        +  where 7 = len({'isLast': False, 'maxResults': 4, 'nextPage': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=4', 'self': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=0', ...})\nE        +    where {'isLast': False, 'maxResults': 4, 'nextPage': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=4', 'self': 'https://neeteshrajoriya3.atlassian.net/rest/api/3/project/search?maxResults=4&startAt=0', ...} = <bound method Response.json of <Response [200]>>()\nE        +      where <bound method Response.json of <Response [200]>> = <Response [200]>.json\n\ntests\\test_project.py:94: AssertionError"}, "attachments": [{"name": "log", "source": "4ca9c41c-af00-40a6-b3e4-53e05451b107-attachment.txt", "type": "text/plain"}], "start": 1743234024818, "stop": 1743234025164, "uuid": "ab8b81f7-d5cb-44ed-bff0-6e596c96af03", "historyId": "fe46fddf00f7926e8d6816474e0c8011", "testCaseId": "fe46fddf00f7926e8d6816474e0c8011", "fullName": "tests.test_project#test_get_projects_paginated", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_project"}, {"name": "host", "value": "user"}, {"name": "thread", "value": "19804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_project"}]}